stdlib/xmlrpc/client.pyi

```diff
@@ -15,7 +15,19 @@ class _SupportsTimeTuple(Protocol):
 
 _DateTimeComparable: TypeAlias = DateTime | datetime | str | _SupportsTimeTuple
 _Marshallable: TypeAlias = (
-    bool | int | float | str | bytes | None | tuple[Any, ...] | list[Any] | dict[Any, Any] | datetime | DateTime | Binary
+    bool
+    | int
+    | float
+    | str
+    | bytes
+    | bytearray
+    | None
+    | tuple[Any, ...]
+    | list[Any]
+    | dict[Any, Any]
```

Comment:
There's a description of marshallable types here, below the paragraph that starts with "Types that are conformable" -- it seems it should be a recursive type: https://docs.python.org/3/library/xmlrpc.client.html#xmlrpc.client.ServerProxy

```suggestion
    | tuple[_Marshallable, ...]
    | list[_Marshallable]
    | dict[str, _Marshallable]
```

Reply:
Using that exact type would be impractical because of invariance. However, we can change tuple, and you're right that the dict keys must be str.
```suggestion
    | tuple[_Marshallable, ...]
    | list[Any]
    | dict[str, Any]
```